content/languages.yml

```diff
@@ -0,0 +1,93 @@
+- label: 'العربية'
+  url: https://ar.reactjs.org
```

Comment:
I for isreacttranslatedyet.com and the scripts I just have:

```
name: Azərbaycanca
code: az
```

And do something like:

```js
const siteName = `${code}.reactjs.org`
const url = `https://${siteName}`
const repository = `https://github.com/reactjs/${siteName}/`
```

We're probably going to enforce that (langs have to be under reactjs, must use lang code) so I don't see why we need the full url?

Reply:
> We're probably going to enforce that (langs have to be under reactjs, must use lang code) so I don't see why we need the full url?

Because I wasn't aware of any such constraints, as I haven't really participated in any of these discussions.

Also, the primary language (English) doesn't have a subdomain. Perhaps we could also add en.reactjs.org and have it redirect to www but in the absence of that constraint, it seemed better to specify the full URL.

Is this really that important? 😄

Reply:
nah, I'm just lazy and don't like typing.